vck/src/commonMain/kotlin/at/asitplus/wallet/lib/agent/Verifier.kt

```diff
@@ -34,7 +34,7 @@ interface Verifier {
     suspend fun verifyPresentationSdJwt(
         input: SdJwtSigned,
         challenge: String,
-        transactionData: Pair<PresentationRequestParameters.Flow, List<TransactionDataBase64Url>>? = null,
```

Comment:
Please keep old method signature and migrate (with deprecation warning) to new method